spotinst/oceancd_verification_template/fields_spotinst_oceancd_verification_provider.go

```diff
@@ -0,0 +1,57 @@
+package oceancd_verification_template
+
+import (
+	"fmt"
+	"github.com/spotinst/spotinst-sdk-go/spotinst"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/spotinst/terraform-provider-spotinst/spotinst/commons"
+)
+
+func Setup(fieldsMap map[commons.FieldName]*commons.GenericField) {
+
+	fieldsMap[Name] = commons.NewGenericField(
+		commons.OceanCDVerificationTemplate,
+		Name,
+		&schema.Schema{
+			Type:     schema.TypeString,
+			Required: true,
+		},
+		func(resourceObject interface{}, resourceData *schema.ResourceData, meta interface{}) error {
+			verificationTemplateWrapper := resourceObject.(*commons.OceanCDVerificationTemplateWrapper)
+			verificationTemplate := verificationTemplateWrapper.GetVerificationTemplate()
+			var value *string = nil
+			if verificationTemplate.Name != nil {
+				value = verificationTemplate.Name
+			}
+			if err := resourceData.Set(string(Name), value); err != nil {
+				return fmt.Errorf(string(commons.FailureFieldReadPattern), string(Name), err)
+			}
+			return nil
+		},
+		func(resourceObject interface{}, resourceData *schema.ResourceData, meta interface{}) error {
+			verificationTemplateWrapper := resourceObject.(*commons.OceanCDVerificationTemplateWrapper)
+			verificationTemplate := verificationTemplateWrapper.GetVerificationTemplate()
+			if value, ok := resourceData.Get(string(Name)).(string); ok && value != "" {
+				verificationTemplate.SetName(spotinst.String(value))
+			}
+			return nil
+		},
+		func(resourceObject interface{}, resourceData *schema.ResourceData, meta interface{}) error {
+			return nil
+		},
+		nil,
+	)
+}
+
+func expandClusterIDs(data interface{}) ([]string, error) {
```

Comment:
why do we need this, please remove this function

Reply:
Removed